Permissions/Core/MvxPermissions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Plugin.Permissions;
+using Plugin.Permissions.Abstractions;
+
+namespace MvvmCross.Plugins.Permissions
+{
+    public class MvxPermissions : IMvxPermissions
+    {
+        public Task<bool> ShouldShowRequestPermissionRationaleAsync(Permission permission)
```

Comment:
No need to create an unnecessary context sync / wrap I believe, just pass the `Task` to the caller.